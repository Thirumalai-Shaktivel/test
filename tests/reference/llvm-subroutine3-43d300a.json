{
    "basename": "llvm-subroutine3-43d300a",
    "cmd": "lfortran --show-llvm {infile} -o {outfile}",
    "infile": "tests/subroutine3.f90",
    "infile_hash": "a524604ef3f17ed8a7b81b5ac3b03932cbc3ec6f308982532b4d9900",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-subroutine3-43d300a.stdout",
    "stdout_hash": "fd8d60bfb8b259191b5fb0245e45ad4db4db6fb3a8469ab35badef34",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}