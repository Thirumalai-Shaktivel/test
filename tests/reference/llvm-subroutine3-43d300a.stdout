; ModuleID = 'LFortran'
source_filename = "LFortran"

define i64 @f() {
EntryBlock:
  ret i64 42
}

